modules/end-user-portal/src/Pages/Apply/Review.cshtml

```diff
@@ -0,0 +1,137 @@
+@page
+@using GpConnect.NationalDataSharingPortal.EndUserPortal.Helpers.Constants
+@using GpConnect.NationalDataSharingPortal.EndUserPortal.Helpers
+@using GpConnect.NationalDataSharingPortal.EndUserPortal.Models
+@using Microsoft.AspNetCore.Mvc.ModelBinding
+@model GpConnect.NationalDataSharingPortal.EndUserPortal.Pages.Apply.ReviewModel
+
+<div class="nhsuk-width-container">
+    <main class="nhsuk-main-wrapper" id="maincontent">
+        <form method="post" id="review-form">
+            <div class="nhsuk-grid-row">
+                <div class="nhsuk-grid-column-full">
+                    <h1 class="nhsuk-label-wrapper">
+                        <label class="nhsuk-label nhsuk-label--xl">Apply for the National Data Sharing Agreement</label>
+                    </h1>
+                    <div class="nhsuk-grid-row">
+                        <div class="nhsuk-grid-column-full">
+                            <h1 class="nhsuk-label-wrapper">
+                                <label class="nhsuk-label nhsuk-label--l">
+                                    Review your answers
+                                </label>
+                            </h1>
+                            <label class="nhsuk-label">Please check that the details you've given are correct before submitting.</label>
+                        </div>
+                    </div>
+                    <div class="nhsuk-grid-row">
+                        <div class="nhsuk-grid-column-full">
+                            <div class="nhsuk-card nhsuk-card--feature">
+                                <div class="nhsuk-card__content nhsuk-card__content--feature">
+                                    <h2 class="nhsuk-card__heading nhsuk-card__heading--feature nhsuk-heading-m">
+                                        Software supplier
+                                    </h2>
+                                    <dl class="nhsuk-summary-list nhsuk-summary-list--no-border">
+                                        <div class="nhsuk-summary-list__row">
+                                        <dt class="nhsuk-summary-list__key">Supplier name:</dt>
+                                        <dd class="nhsuk-summary-list__value">@Model.SupplierName</dd>
+                                        <dd class="nhsuk-summary-list__actions"><a href="./SoftwareSupplier">Change<span class="nhsuk-u-visually-hidden"> name</span></a></dd>
+                                </div>
+                                <div class="nhsuk-summary-list__row">
+                                    <dt class="nhsuk-summary-list__key">GP Connect product(s):</dt>
+                                    <dd class="nhsuk-summary-list__value">
+                                        @foreach (var softwareSupplierProduct in Model.SupplierProducts)
```

Comment:
The terminology here is confusing, I thought this was talking about the selected vendor not the GP connect interactions.

Reply:
This has been updated as per the other comment.